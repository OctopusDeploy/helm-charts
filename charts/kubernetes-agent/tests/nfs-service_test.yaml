suite: "persistence"
templates:
- templates/nfs-service.yaml
tests:
- it: "should match snapshot"
  asserts:
  - matchSnapshot: {}
