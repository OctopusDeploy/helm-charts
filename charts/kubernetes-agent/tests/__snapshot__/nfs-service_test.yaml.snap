should match snapshot:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      name: octopus-agent-nfs
      namespace: NAMESPACE
    spec:
      clusterIP: None
      ports:
        - name: nfs
          port: 2049
          protocol: TCP
      selector:
        app.kubernetes.io/name: octopus-agent-nfs
